packages/hash/frontend/src/pages/_document.page.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import createEmotionServer from "@emotion/server/create-instance";
+import withTwindDocument from "@twind/next/document";
+import Document, { DocumentContext } from "next/document";
+
+import twindConfig from "../../twind.config";
+import { createEmotionCache } from "../theme";
+
+class MyDocument extends Document {
+  static async getInitialProps(context: DocumentContext) {
+    const initialProps = await Document.getInitialProps(context);
+    return { ...initialProps };
+  }
+}
+
+// `getInitialProps` belongs to `_document` (instead of `_app`),
+// it's compatible with static-site generation (SSG).
+MyDocument.getInitialProps = async (ctx) => {
```

Comment:
I think we should indicate in a comment that we're basing this configuration off an existing example (from memory this was https://github.com/mui/material-ui/blob/master/examples/nextjs/pages/_document.js)

Reply:
Added in 756940